craft_cli/messages.py

```diff
@@ -51,30 +51,30 @@ def get_terminal_width() -> int:
 
 
 def get_log_filepath(appname: str) -> pathlib.Path:
-    """Provide a filepath for logging into.
+    """Provide a unique filepath for logging.
 
     The app name is used for both the directory where the logs are located and each log name.
 
     Rules:
     - use an appdirs provided directory
     - base filename is <appname>.<timestamp with microseconds>.log
-    - it rotates until it gets to ROTATION_LIMIT
+    - it rotates until it gets to reaches :data:`._MAX_LOG_FILES`
```

Comment:
I would make this public as it is documented

Reply:
As we talked, made everything private.